build.gradle

```diff
@@ -148,3 +148,29 @@ configurations.all {
     resolutionStrategy.force 'commons-codec:commons-codec:1.13'
     resolutionStrategy.force 'com.google.guava:guava:31.0.1-jre'
 }
+
+// versionIncrement: Task to auto increment to the next development iteration
+task versionIncrement {
+    onlyIf { System.getProperty('newVersion') }
+    doLast {
+        ext.newVersion = System.getProperty('newVersion')
+        println "Setting version to ${newVersion}."
+        // String tokenization to support -SNAPSHOT
+        ant.replaceregexp(match: opensearch_version.tokenize('-')[0], replace: newVersion.tokenize('-')[0], flags:'g', byline:true) {
+            fileset(dir: projectDir) {
+                // Include the required files that needs to be updated with new Version
+                include(name: ".github/workflows/sql-workbench-test-and-build-workflow.yml")
+                include(name: "doctest/build.gradle")
+                include(name: "build.gradle")
+            }
+        }
+        ant.replaceregexp(file:'sql-jdbc/build.gradle', match:"version '\\d+.\\d+.\\d+.\\d+'", replace:"version " + "'" + newVersion.tokenize('-')[0] + '.0' + "'", flags:'g', byline:true)
```

Comment:
could you remove this line? sql-jdbc was accidentally bumped in previous PR, but it's a driver and the version should be independent

Reply:
Pushed in my latest commit @joshuali925

Reply:
thanks. the latest sql-jdbc release is `1.1.0.1`, the next one should be `1.2.0.0` or `2.0.0.0` depending on the changes, but should not be the same version as core and plugins